tests/network-tests/src/fixtures/membership/VerifyValidatorMembershipFixture.ts

```diff
@@ -49,38 +48,23 @@ export class VerifyValidatorMembershipFixture extends WithMembershipWorkersFixtu
     }
   }
 
-  private assertQueriedMembershipsAreValid(
-    qMembers: MembershipFieldsFragment[],
-    qEvents: ValididatorVerificationStatusUpdatedEventFieldsFragment[]
-  ): void {
-    const verifiedSuccessfully = this.verifications.filter((m) => !m.expectFailure).length
-    assert.equal(qEvents.length, verifiedSuccessfully, 'Too many validator memberships were verified')
-
+  private assertQueriedMembershipsAreValid(qMembers: MembershipFieldsFragment[]): void {
     this.events.map((e, i) => {
       const verification = this.verifications[i]
       if (verification.expectFailure) return
 
-      const qMembership = qMembers.find((p) => p.id === verification.memberId)
+      const qMembership = qMembers.find((p) => p.id === verification.memberId.toString())
       Utils.assert(qMembership, 'Query node: Membership not found')
     })
   }
 
-  protected assertQueryNodeEventIsValid(qEvent: ValididatorVerificationStatusUpdatedEventFieldsFragment, i: number): void {
-    const { memberId, isVerified, asWorker } = this.verifications[i]
-    assert.equal(qEvent.member.id, memberId)
-    assert.equal(qEvent.actor.id, `membershipWorkingGroup-${asWorker ? asWorker.toString() : this.membershipLeadId!.toString()}`)
-    assert.equal(qEvent.metadata.is_verified, isVerified)
+  protected assertQueryNodeEventIsValid(qEvent: AnyQueryNodeEvent, i: number): void {
```

Comment:
@zeeshanakram3 as this method must be defined, should the event be `MemberProfileUpdatedEvent` or should it be a new event specific to this action (e.g `ValididatorVerificationStatusUpdatedEvent`), or should this method remain empty ?